layers/ip6.go

```diff
@@ -34,6 +75,7 @@ type IPv6 struct {
 	HopLimit     uint8
 	SrcIP        net.IP
 	DstIP        net.IP
+	RoutingDstIP net.IP
```

Comment:
If I understand correctly RoutingDst() will look ahead interpreting the IPv6 layer payload bytes until and if we find an IPv6 routing extension header?